tuned/utils/config_parser.py

```diff
@@ -0,0 +1,121 @@
+# Extension of ConfigParser for python 2.7
+# After support for p 2.7 is dropped, so can be this file
+
+try:
+	import ConfigParser as cp
+	from StringIO import StringIO
+except ImportError:
+	raise ImportError("This file should not be used in python 3")
+import re
+
+SECTION_REGEX = r'''^
```

Comment:
I think we could simplify things here and for the 2.7 just emulate the inline comments after parsing the file with the re.sub() on the parsed values. I think it won't be problem if the parser doesn't work 100% as with the python 3.